server/chat-plugins/helptickets.ts

```diff
@@ -1057,14 +1109,14 @@ export const commands: ChatCommands = {
 			if (!user.named) return this.popupReply(this.tr`You need to choose a username before doing this.`);
 			const banMsg = HelpTicket.checkBanned(user);
 			if (banMsg) return this.popupReply(banMsg);
+			const tickets = getTickets();
 			let ticket = tickets[user.id];
```

Comment:
```suggestion
			let helpRoom = Rooms.get(`help-${user.id}`);
			let ticket = helpRoom?.ticket;
```

Reply:
(There's more code needed to make this work, but you should be able to take it from here.)